pkg/u-boot/rpi/config.txt

```diff
@@ -25,7 +25,7 @@ avoid_warnings=1
 
 # This overlay fixes divergences between the RPi device tree and what
 # upstream provides
```

Comment:
Is this comment still correct given that you changed the overlay?

Reply:
@eriknordmark No, it's not correct, I missed it, I'll change the comment.